torch/_inductor/compile_fx.py

```diff
@@ -227,16 +232,14 @@ def materialize(x):
                         if param is not None
                     ]
                     real_inputs = [
-                        materialize(x) for x in [*params_flat, *V.real_inputs]
+                        materialize(x) for x in (params_flat + V.real_inputs)
                     ]
                 else:
                     real_inputs = [materialize(x) for x in V.real_inputs]
 
                 with torch.utils._python_dispatch._disable_current_modes():
                     compiled(real_inputs)
 
-                real_inputs = None
```

Comment:
this might be for memory reasons. change it to `del real_inputs` instead ? we should address this before landing.

Reply:
Good call. Fixed. Thanks.